.github/workflows/housekeeping.yaml

```diff
@@ -21,3 +22,19 @@ jobs:
           days-before-pr-close: 7
           stale-pr-label: 'Stale'
 
+  branches:
+      name: Cleanup old branches
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v2
+        - name: Run delete-old-branches-action
+          uses: beatlabs/delete-old-branches-action@v0.0.9
+          with:
+            repo_token: ${{ github.token }}
+            date: '2 months ago'
+            dry_run: false
+            delete_tags: true
```

Comment:
what does `delete_tags` mean? We should never delete git tags.

Reply:
oops true story.... What it does is it treats all tags as branches -- meaning it does delete all tags older than 2 months. I am disabling that for now but we can consider performing a tag clean up using the 
`extra_protected_tag_regex` in order to protect only release tags (we rarely perform a `git push --tags` ...  but our open-source community can do so right?)